content/_index.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Home"
++++
+
+
+# MRC Centre for Global Infectious Disease Analysis
+
+
+These pages contain an overview of the software output of the
+department. Software has been categorised as either [research code](/research)
+(directly authored by researchers) or [tools](/tools) (primarily authored by research 
+software engineers). There is a high degree of inter-connection between the 
+various tools and research outputs. To illustrate 
+the ways in which research and tools interact, see the graphs listed on the 
+[connections](/connections) page.
+
```

Comment:
how does the summary graph end up on this page?

Reply:
Via the index layout: https://github.com/mrc-ide/mrc-ide.github.io/pull/3/files#diff-96762550561f76f989ced99c4a0751e94696682490ee94b2d77a66377c9619ebR12

could be moved here though